packages/legacy/src/queries/tree.ts

```diff
@@ -128,3 +142,32 @@ export const useInventoryTreeQuery = <T extends InventoryTree>(
     apiMockData as T
   );
 };
+
+function createTreeFromVMs(vms: ICommonTreeObject[]): InventoryTree {
```

Comment:
cc:// @bkhizgiy - this is using the `/vms` endpoint to create a `/tree/vms`  API response